apps/studio/components/interfaces/APIKeys/CreateNewAPIKeysButton.tsx

```diff
@@ -49,16 +50,16 @@ export const CreateNewAPIKeysButton = () => {
         <AlertDialogHeader>
           <AlertDialogTitle>Create new API keys</AlertDialogTitle>
           <AlertDialogDescription>
-            This will create a default publishable key and a default secret key named{' '}
+            This will create a default publishable key and a default secret key both named{' '}
             <code className="!break-keep text-code-inline">default</code>. These keys are required
             to connect your application to your Supabase project.
           </AlertDialogDescription>
         </AlertDialogHeader>
         <AlertDialogFooter>
           <AlertDialogCancel>Cancel</AlertDialogCancel>
-          <Button onClick={handleCreateNewApiKeys} loading={isCreatingKeys}>
+          <AlertDialogAction onClick={handleCreateNewApiKeys} disabled={isCreatingKeys}>
```

Comment:
Follows [the Radix action button](https://www.radix-ui.com/primitives/docs/components/alert-dialog#action) as every other `AlertDialogFooter` does.

Disadvantage: loses the ability to show _loading_. Perhaps thats okay here? I can’t see a loading state in my testing. It’s too quick.

Or should we revert to `Button`?

Reply:
Agree and noted. Thanks!